Makefile.prow

```diff
@@ -57,12 +57,7 @@ lint:
 
 .PHONY: lint-all
 
-lint-all: lint-yaml lint-go
```

Comment:
Sure we don't know to lint the yaml files?

Reply:
My mistake, we still do want to lint the yaml files. I was just having an issue with pip install on linux as the executables are not referenced unless I add to path like so: `export PATH="$HOME/.local/bin:$PATH"`